src/main/java/edu/virginia/lib/fedora/aview/AView.java

```diff
@@ -77,12 +101,21 @@ public Response listAccessionsHTML(@Context UriInfo uriInfo) throws IOException,
         VelocityContext context = new VelocityContext();
         context.put("accessionRoot", getASpaceURI(uriInfo));
         context.put("helper", helper);
-        context.put("accessions", getTriplestore(uriInfo).getQueryResponse("PREFIX fcrepo: <http://fedora.info/definitions/v4/repository#>\n" +
-                "\n" +
-                "SELECT ?a \n" +
-                "WHERE {\n" +
-                "    ?a fcrepo:hasParent <http://" + uriInfo.getRequestUri().getHost() + ":8080/fcrepo/rest/aspace>\n" +
-                "}"));
+        
+        if (host == null)
+        	context.put("accessions", getTriplestore(uriInfo).getQueryResponse("PREFIX fcrepo: <http://fedora.info/definitions/v4/repository#>\n" +
+                    "\n" +
+                    "SELECT ?a \n" +
+                    "WHERE {\n" +
+                    "    ?a fcrepo:hasParent <http://" + uriInfo.getRequestUri().getHost() + ":8080/fcrepo/rest/aspace>\n" +
```

Comment:
(host == null ? uriInfo.getRequestUri().getHost() : host) instead of uriInfo.getRequestUri().getHost()